core/src/main/java/sem/group15/bubblebobble/core/LevelRenderer.java

```diff
@@ -107,12 +105,36 @@ public void renderLost(int currentLevelNumber) {
         Gdx.gl.glClearColor(0, 0, 0, 0);
         Gdx.gl.glClear(GL30.GL_COLOR_BUFFER_BIT);
         batch.begin();
-        renderWorld();
+        renderWorld(this.level);
         Gdx.gl.glBlendColor(1, 1, 1, 0.5f);
         String str = "You made it to level " + currentLevelNumber
                 + "\nScore: " + level.getPlayer().score
                 + "\n\nPress enter to start a new game!";
         font.drawMultiLine(batch, str, Gdx.graphics.getWidth() / 2, Gdx.graphics.getHeight() / 2);
+        draw();
+    }
+
+    private void draw() {
+        draw(0);
+    }
+
+    private void draw(float verticalOffset) {
+        batch.setTransformMatrix((new Matrix4()).setTranslation(0,verticalOffset, 0));
         batch.end();
     }
```

Comment:
Why have draw() that just calls draw(0), instead of just using draw(0) everywhere? 
<review